holidays/countries/zambia.py

```diff
@@ -27,70 +28,77 @@ class Zambia(HolidayBase):
     """
 
     country = "ZM"
+    special_holidays = {
+        2016: (
+            (AUG, 11, "General elections and referendum"),
+            (
+                SEP,
+                13,
+                "Inauguration ceremony of President-elect "
+                "and Vice President-elect",
+            ),
+        ),
+        2018: (
+            (MAR, 9, "Public holiday"),
+            (JUL, 26, "Lusaka mayoral and other local government elections"),
+        ),
+        2021: (
+            (JUL, 2, "Memorial service for Kenneth Kaunda"),
+            (JUL, 7, "Funeral of Kenneth Kaunda"),
+            (AUG, 12, "General elections"),
+            (AUG, 13, "Counting in general elections"),
+            (AUG, 24, "Presidential inauguration"),
+        ),
+        2022: ((MAR, 18, "Funeral of Rupiah Banda"),),
+    }
 
     def _populate(self, year):
         super()._populate(year)
```

Comment:
I'd move it below the line 60.

Reply:
> I'd move it below the line 60.

`special_holidays` declaration?

Reply:
Nope :).
I see the weird code selection here too, my bad. 

I meant `super()._populate(year)` line. It should be executed after the year condition check is satisfied.

Reply:
Done.